dbus_service.py

```diff
@@ -85,7 +85,7 @@ def __init__(
         else:
             self._read_config_template(actual_inverter)
 
-        logging.debug("%s /DeviceInstance = %d", servicename, self.deviceinstance)
+        logging.critical("%s /DeviceInstance = %d", servicename, self.deviceinstance)
```

Comment:
same

Reply:
agree, but it shows at a minimum level und only at startup, that the service is started and what is registered, and only at startup